internal/commands/scan.go

```diff
@@ -973,35 +974,57 @@ func addAPISecScan(cmd *cobra.Command) map[string]interface{} {
 	}
 	return nil
 }
-
-func addSCSScan(cmd *cobra.Command) (map[string]interface{}, error) {
-	if scanTypeEnabled(commonParams.ScsType) {
+func createResubmitConfig(resubmitConfig []wrappers.Config, scsRepoToken, scsRepoURL string) wrappers.SCSConfig {
+	scsConfig := wrappers.SCSConfig{}
+	for _, config := range resubmitConfig {
+		resubmitTwoms := config.Value[configTwoms]
```

Comment:
what is the difference between the regular config building and resubmit? Can't we re-use the regular build config?